### ISSUE TITLE
Extracting premake-5.0.0-beta2-linux.tar.gz will break the ownership of the extract target dir

### ISSUE THREAD

**Issue by mercury233** [premake/premake-core#1966]
_Before opening a new bug report, please read [Reporting Bugs](https://github.com/premake/premake-core/blob/master/CONTRIBUTING.md#reporting-bugs) and consider if this is something [you can contribute](https://github.com/premake/premake-core/blob/master/CONTRIBUTING.md#contributing-a-fix-or-feature) yourself. If this a new bug, help us help you by filling in the template below._

**What seems to be the problem?**
The `premake-5.0.0-beta2-linux.tar.gz` includes "." in its files, extracting it with tar will set the permission and owner of the directory accidentally. If you extracted at the root dir like me, you will found your root dir got own by some other user, and ssh won't let you connect to this server.
> Authentication refused: bad ownership or modes for directory /root

**What did you expect to happen?**

**What have you tried so far?**

**How can we reproduce this?**
https://github.com/mercury233/test-premake/actions/runs/3141390884/jobs/5103770163#step:2:4
As you can see, the permission of test-ownership changed from `0755/drwxr-xr-x` to `0700/drwx------` after extracting.

**What version of Premake are you using?**

**Anything else we should know?**

**Comment by samsinsane** [comment]
@nickclark2016 looks like the packaging script might need some modification? Probably needs to execute the packing step outside of the folder with the Premake executable, instead of inside it.

**Comment by samsinsane** [comment]
Looks like the macOS archive might have the same problem. Additionally, the Linux binary is a `-dev` build not a `-beta2` build, I imagine the macOS binary is the same?

**Comment by nickclark2016** [comment]
@mercury233 Can you confirm if this is still an issue in the latest release packages?

**Comment by mercury233** [comment]
> [@mercury233](https://github.com/mercury233) Can you confirm if this is still an issue in the latest release packages?

I think the beta5 release don't have this issue

**Comment by mercury233** [comment]
@nickclark2016 Shouldn't the file be re-uploaded? premake-5.0.0-beta2-linux.tar.gz I downloaded yesterday still has this problem

**Comment by nickclark2016** [comment]
I don't believe we do. Changing build artifacts can be considered a security concern. It is better to just update to use the latest binaries.
